test-data/unit/check-newsemanal.test

```diff
@@ -1476,6 +1476,8 @@ strict_optional = True
 [[mypy-b]
 strict_optional = False
 
+-- ]]
```

Comment:
Btw, why is this needed? We a bunch of unbalanced `[` in other test files.

Reply:
Yeah, and I usually fix them when I run into them :P 
They mess up editor indentation, which I find super annoying